lib/msal-browser/docs/logging.md

```diff
@@ -0,0 +1,60 @@
+# MSAL JS logging instructions
+
+These are steps needed to enable MSAL JS logging for your applications:
+
+1. In the MSAL [Configuration](./configuration.md) object, you can enable logging to collect msal js logs. We enable different levels of logging and an appropriate level can be chosen as needed. 
+
+2. The logger options can be set as below, the example chooses `LogLevel.Trace`
+
+```javascript
+
```

Comment:
```suggestion
import { PublicClientApplication, LogLevel } from "@azure/msal-browser";
```

Reply:
Ah! I meant in the actual sample. This can be added here, yes.

Reply:
Ah ok, sorry. I think we have both variety in the samples (vanilla js samples uses the msal namespace, while react and angular samples import things ES6 style)